src/vmm/src/cpu_config/x86_64/cpuid/normalize.rs

```diff
@@ -237,62 +237,51 @@ impl super::Cpuid {
             .get_mut(&CpuidKey::leaf(0x1))
             .ok_or(FeatureInformationError::MissingLeaf1)?;
 
-        // CLFLUSH line size (Value ∗ 8 = cache line size in bytes; used also by CLFLUSHOPT).
-        //
-        // clflush: 8..16,
+        // CPUID.01H:EBX[15:08]
```

Comment:
mh, so intel has the upper bound inclusive? In that case: nit: let's use rust inclusive ranges below as well, e.g. `8..=15` instead of `8..16` (same for all the others), so that the comments and code continue matching

Reply:
yeah, Intel doc uses the upper bound inclusive. I tried to use inclusive ranges for it, but it didn't work out of the box and I gave it up quickly :P Thanks for pushing me to do that!! will do!